pygameMenu/widgets/widget.py

```diff
@@ -312,6 +312,29 @@ def set_selected(self, selected=True):
         :return: None
         """
         self.selected = selected
+        if selected:
+            self._focus()
+        else:
+            self._blur()
+
+        # Sync events with the system
+        _pygame.event.pump()
```

Comment:
What the purpose of `pump()`?
What happen if not present?

Reply:
That was a really interesting bug, if you remove the .pump() call somehow if you keep pressed a button the pygame.events list grows; if you lose the focus (changing to other widget) the key you last pressed keeps active until you press/release it (I really don't know how to easily explain this), maybe if you lose the focus for a couple of miliseconds the "key_release_ms" is not applied, saturating the input.

Looking through internet I found that .pump() method cheks all the input and remaps the active keys. That fix the bug because the "immense" event list reduces a lot